symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkCoreConfig.java

```diff
@@ -44,8 +47,14 @@ public ApiClient datafeedAgentApiClient(ApiClientFactory apiClientFactory) {
   }
 
   @Bean(name = "podApiClient")
-  public ApiClient podApiClient(ApiClientFactory apiClientFactory) {
-    return apiClientFactory.getPodClient();
+  public ApiClient podApiClient(ApiClientFactory apiClientFactory, @Nullable AuthSession authSession, BdkConfig config) {
+    ApiClient client = apiClientFactory.getPodClient();
+    if (config.getCommonJwt().getEnabled()) {
+      final OAuthSession oAuthSession = new OAuthSession(authSession);
+      client.getAuthentications().put("bearerAuth", new OAuthentication(oAuthSession::getBearerToken));
+      client.addEnforcedAuthenticationScheme("bearerAuth");
```

Comment:
This could be tested in `SymphonyBdkAutoConfigurationTest`

Reply:
Done https://github.com/finos/symphony-bdk-java/pull/618/commits/038daede4e4e37dd03eb617f26c70aaa8af53c18